clang/lib/Sema/HeuristicResolver.cpp

```diff
@@ -315,9 +315,9 @@ std::vector<const NamedDecl *> HeuristicResolverImpl::resolveMemberExpr(
 
 std::vector<const NamedDecl *>
 HeuristicResolverImpl::resolveDeclRefExpr(const DependentScopeDeclRefExpr *RE) {
-  return resolveDependentMember(
-      resolveNestedNameSpecifierToType(RE->getQualifier()), RE->getDeclName(),
-      StaticFilter);
+  QualType Qualifier = resolveNestedNameSpecifierToType(RE->getQualifier());
+  Qualifier = simplifyType(Qualifier, nullptr, /*UnwrapPointer=*/false);
```

Comment:
```suggestion
  Qualifier = simplifyType(Qualifier, /*E=*/nullptr, /*UnwrapPointer=*/false);
```

The name is not the most descriptive.

Reply:
Done.

(`Expr` would be a better name, but that is a type name.)